### PR TITLE
fix: confirm support for flake8 v6

### PR DESCRIPTION
**Pull request by JanMalte**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
 python = "^3.7"
-flake8 = ">=3.8.4,<6"
+flake8 = ">=3.8.4,<7"
```
Comment (pawelad) [inline pyproject.toml:27]: I'd suggest removing the upper constraint altogether (see https://github.com/rocioar/flake8-django/issues/104).

**Review by pawelad — APPROVED** [review]
👍

**Comment by washeck** [comment]
@rocioar Could you please have a look at this? It is creating a dependency hell for us between pyflakes, flake8 and several flake8 plugins.

**Comment by mredaelli** [comment]
@rocioar Anything against this?

**Pull request merged**